client/components/Editors/Portal/Sections/Github.tsx

```diff
@@ -23,10 +24,11 @@ export default function GithubSection() {
 function User() {
   const user = useStore((state) => state.descriptor.github?.user)
   const updateGithub = useStore((state) => state.updateGithub)
+  const { t } = useTranslation()
```

Comment:
is there a way to make `t` global so we dont have to import it every time? Is it worth it? What do you think @roll

Reply:
No it's not possible as `useTranslation` will make react to rerender when the language changes